imagej/imagej-ops2-tutorial/docs/MigratingFromImageJOps.md

```diff
@@ -0,0 +1,198 @@
+# Migrating from ImageJ Ops
+
+The [ImageJ Ops](https://imagej.net/libs/imagej-ops/index) framework is the predecessor of the SciJava Ops framework, and many steps were taken to make expressing Ops much easier. This document shows ImageJ Ops developers how to convert their Ops to SciJava Ops - we'll convert [`DefaultGaussRAI`](https://github.com/imagej/imagej-ops/blob/master/src/main/java/net/imagej/ops/filter/gauss/DefaultGaussRAI.java) from the ImageJ Ops repository as an example.
+
+## 1. Upgrading pom-scijava
```

Comment:
A "Migrating from ImageJ Ops" section is great. Presumably this step is a requisite for any implementer of ops. So if we're going to document migration we might as well have a page about implementation - then this pom-scijava info can be migrated there and reference from the migration doc.

Reply:
@hinerm I wrote a start in 2ea41a72.

I'd like to write an example project that we can direct people to, although that may come along with the work that we're doing in other PRs. I'd like to get those merged (#73, #79, etc) first.